packages/rancher-monitoring/overlay/app-README.md

```diff
@@ -1,10 +1,15 @@
-- [[rancher-pushprox](https://github.com/rancher/charts/tree/dev-v2.5/packages/rancher-pushprox/charts)] Sets up default Deployments and DaemonSets to monitor `kube-scheduler`, `kube-controller-manager`, `kube-proxy`, and `kube-etcd` components via nodeSelectors / tolerations for certain cluster types
-- [[Prometheus Adapter](https://github.com/helm/charts/tree/master/stable/prometheus-adapter)] Exposes custom metrics, resource metrics, and external metrics on the default [Prometheus](https://prometheus.io/) instance launched by [Prometheus Operator](https://github.com/coreos/prometheus-operator)
+ This chart is based off of the upstream [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) chart. The chart deploys [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) and its CRDs along with [Grafana](https://github.com/grafana/helm-charts/tree/main/charts/grafana), [Prometheus Adapter](https://github.com/prometheus-community/helm-charts) and additional charts / Kubernetes manifests to gather metrics. It allows users to monitor their Kubernetes clusters, view metrics in Grafana dashboards, and set up alerts and notifications.
 
-For more information, see the README.md of this chart.
+For more information on how to use the feature, refer to our [docs](https://rancher.com/docs/rancher/v2.x/en/monitoring-alerting/v2.5/).
+
+The chart installs the following components:
+
+- [Prometheus Operator](https://github.com/coreos/prometheus-operator)  - The operator provides easy monitoring definitions for Kubernetes services, manages [Prometheus](https://prometheus.io/) and [AlertManager](https://prometheus.io/docs/alerting/latest/alertmanager/) instances, and adds default scrape targets for some Kubernetes components.
+- [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus/) - a collection of community-curated Kubernetes manifests, Grafana Dashboards, and PrometheusRules that deploy a default end-to-end cluster monitoring configuration.
+- [Grafana](https://github.com/helm/charts/tree/master/stable/grafana) - Grafana allows the ability to view dashboards of the Kubernetes cluster. The chart installs several default dashboards based on metrics available in the cluster.
+- [node-exporter](https://github.com/helm/charts/tree/master/stable/prometheus-node-exporter)/[kube-state-metrics](https://github.com/helm/charts/tree/master/stable/kube-state-metrics)/[rancher-pushprox](https://github.com/rancher/charts/tree/dev-v2.5/packages/rancher-pushprox/charts) - These components set up monitoring of internal Kubernetes components across different Kubernetes cluster types.
+- [Prometheus Adapter](https://github.com/helm/charts/tree/master/stable/prometheus-adapter) - The adapter exposes custom metrics, resource metrics, and external metrics on the default [Prometheus](https://prometheus.io/) instance.
```

Comment:
nit: we don't expose metrics by default, but a user has the ability to configure that with this deployed. By default, we only expose [certain container-based custom metrics](https://github.com/rancher/charts/blob/1d17a5bb6767f6468815b005b688fb73f7baa228/charts/rancher-monitoring/charts/prometheus-adapter/templates/custom-metrics-configmap.yaml#L16-L81).
```suggestion
- [Prometheus Adapter](https://github.com/helm/charts/tree/master/stable/prometheus-adapter) - The adapter allows a user to expose custom metrics, resource metrics, and external metrics on the default [Prometheus](https://prometheus.io/) instance to the Kubernetes API Server.
```